types/2020-08-27/PaymentIntents.d.ts

```diff
@@ -787,7 +791,7 @@ declare module 'stripe' {
         /**
          * Type of the payment method for which payment is in `processing` state, one of `card`.
          */
-        type: string;
+        type: 'card';
```

Comment:
This change conversion from string to enum is fine since we haven't actually released https://github.com/stripe/stripe-node/pull/1312 yet.